packages/ui-kit/src/components/mermaid-lazy.tsx

```diff
@@ -0,0 +1,16 @@
+import { lazy } from 'react';
+
+const mermaidVersion = '9.3';
+
+const Mermaid = lazy(() => {
+  return new Promise((resolve) => {
+    const script = document.createElement('script');
+    script.src = `https://cdn.jsdelivr.net/npm/mermaid@${mermaidVersion}/dist/mermaid.min.js`;
+    script.onload = resolve;
+    document.body.appendChild(script);
+  }).then(() => {
+    return import('./mermaid-client-side');
+  });
```

Comment:
@emmenko  I did not use gatsbys `<Script>` because it would load that big chunk for everybody on the first page hit, not only when they hit one of the very few pages that have such a diagram.  I was looking for something that only loads it when needed like the current implementation does.